bump_version.sh

```diff
@@ -1,3 +1,35 @@
+#!/bin/bash
 
+###
+# EXIT CODES
+# 0 - success
+# 1 - parameter error
+###
 
+# Sample environment variables
+# DOCKERHUB_REPO="pavics/weaver"
+# BUMP_TAG="WEAVER_WORKER_IMAGE"
+# BUMP_TAG_VALUE="1.13.2-worker"
+# BUMP_FILE="birdhouse/default.env"
 
+REQUIRED_ENV_VARS='
+    DOCKERHUB_REPO
+    BUMP_TAG
+    BUMP_TAG_VALUE
+    BUMP_FILE
+'
+
+# args parsing
+for env_var in $REQUIRED_ENV_VARS
+do
+    if [[ ! -v "${env_var}" ]]; then
+        echo "[ERROR] [$0] Missing ${env_var} environment variable. Exiting."
+        exit 1
+    fi
+done
+
+
+NEW_IMAGE_TAG_VALUE=$DOCKERHUB_REPO":"$BUMP_TAG_VALUE
```

Comment:
I think `BUMP_TAG_VALUE` could be renamed to `NEW_TAG_VALUE` to be less confusing vs `BUMP_TAG` which is the variable name in `default.env`.

Reply:
Good suggestion, done.